OpenSim/Tools/Test/sphere.vtp

```diff
@@ -0,0 +1,883 @@
+<?xml version="1.0"?>
+<VTKFile type="PolyData" version="0.1" byte_order="LittleEndian" compressor="vtkZLibDataCompressor">
```

Comment:
Sure, will do. Though keep in mind the test will only be visual since the standard actually compares transforms rather than actual display, so we'll see nothing in the code base to reflect this other than the change of the file name. I may add a png. You'll have to trust me ;)